Cargo.toml

```diff
@@ -30,6 +30,7 @@ appendlist = "1.4"
 ash = { version = "0.37.1", optional = true }
 bitflags = "2.2.1"
 calloop = "0.12.2"
+cursor-icon = "1.0.0"
```

Comment:
This is a winit crate used mostly in clients now, it's used by sctk for its themed cursor as well.

Reply:
The crate notes in it's documentation, that it aims to be cross-platform. Is there reason to assume it might not include all the cursor variants the wayland protocol does at some point to keep it cross-platform? In that case I would assume it would be better to define our own enum.

Reply:
@Drakulix the crate targets w3c specification, which wayland also targets. So it should be fine. The reason for it as well, is that you can call `name()` on icon to load themes, etc.

Reply:
Great, then it is a good fit.